release_docs/RELEASE_PROCESS.md

```diff
@@ -0,0 +1,159 @@
+    1. Verify interface additions, changes, and removals, and update the shared library interface version number.
+    2. Execute the CI snapshot workflow.
+        - Actions - “[hdf5 release build](https://github.com/HDFGroup/hdf5/blob/develop/.github/workflows/release.yml)” workflow and use the defaults.
+    3. Download and inspect release build source and binary files.  Downloaded source files should build correctly, one or more binaries should install and run correctly.  There should be nothing missing nor any extraneous files that aren’t meant for release.
+    4. Verify the interface compatibility reports between the current source and the previous release on the Github [Snapshots](https://github.com/HDFGroup/hdf5/releases/tag/snapshot-1.14) page.
+        - The compatibility reports are produced by the CI and are viewable in the Github [Releases/snapshot](https://github.com/HDFGroup/hdf5/releases/tag/snapshot) section.
+    5. Verify the interface compatibility reports between the current source and the previous release on the Github [Snapshots](https://github.com/HDFGroup/hdf5/releases/tag/snapshot-1.14) page.
+        - The compatibility reports are produced by the CI and are viewable in the Github [Releases/snapshot](https://github.com/HDFGroup/hdf5/releases/tag/snapshot) section.
+    6. Confirm the necessity of and approve of any interface-breaking changes. If any changes need to be reverted, task the developer who made the change to do so as soon as possible. If a change is reverted, return to the previous step and regenerate the compatibility report after the changes is made. Otherwise, continue to the next step.
+    7. Update the .so version numbers in the [config/lt_vers.am](https://github.com/HDFGroup/hdf5/blob/develop/config/lt_vers.am) file in the support branch according to [libtool's library interface version](https://www.gnu.org/software/libtool/manual/libtool.html#Versioning) scheme. 
+        - See [Updating version info (Libtool)](https://www.gnu.org/software/libtool/manual/html_node/Updating-version-info.html#Updating-version-info) for rules to help update library version numbers. 
+    8. After the release branch has been created, run `./autogen.sh` to regenerate build system files on the release branch and commit changes.    
+
+6. Prepare Release Branch (Release Manager)
+    1. Get the release branch ready for pre-release testing and packaging.
+    2. For all release preparation operations, the release coordinator will clone and push directly to canonical HDF5:
+        - `$ git clone ​https://github.com/HDFGroup/hdf5.git`
+    3. Create a release preparation branch (hdf5_X_Y_Z) by branching off of the support branch (hdf5_X_Y):
+        - `$ git checkout hdf5_X_Y`
+        - `$ git checkout -b hdf5_X_Y_Z`
+        - or create the new branch in GitHub GUI.
+    4. Check that required CMake files to point to the specific versions of the third-party software (szip and zlib) that they depend on.
```

Comment:
Check that required CMake files point to...

Reply:
Fixed.